doc/man1/openssl-verification-options.pod

```diff
@@ -18,31 +18,55 @@ L<X509_verify_cert(3)> verify X.509 certificates. The details of how each
 command handles errors are documented on the specific command page.
 
 Certificate verification is a complicated process, consisting of
-a number of separate steps that are detailed in the following paragraphs.
+a number of separate steps that are detailed in the following sections.
+
+=head2 Certification Path Building
 
 First, a certificate chain is built up starting from the target certificate
 and typically ending in a self-signed "root" CA certificate.
 It is an error if the whole chain cannot be built up
 unless the B<-partial_chain> option is given.
+
 The chain is built up iteratively, looking up in turn
-the certificate of the signer ("issuer") of the current certificate.
+a currently valid (non-expired) certificate with suitable key usage that
+matches as an issuer of the current "subject" certificate as described below.
+In each iteration the first certificate found this way is taken; for effciency,
```

Comment:
typo: effciency

Reply:
Thx; fixed the typo and extended the sentence.